docs/en/sql-reference/functions/other-functions.md

```diff
@@ -2138,3 +2138,54 @@ Result:
 
 -   [tcp_port](../../operations/server-configuration-parameters/settings.md#server_configuration_parameters-tcp_port)
 
+## currentProfiles {#current-profiles}
+
+Returns a list of the current [settings profiles](../../operations/access-rights.md#settings-profiles-management) for the current user. 
+
+The settings profile can be specified at the user's definition (see [CREATE USER](../../sql-reference/statements/create/user.md#create-user-statement), `PROFILE` clause) and can be changed with the [SET profile](../../sql-reference/statements/set.md#query-set) statements lately. 
+
+**Syntax**
+
+``` sql
+currentProfiles()
+```
+
+**Returned value**
+
+-   List of the current user settings profiles. 
+
+Type: [Array](../../sql-reference/data-types/array.md)([String](../../sql-reference/data-types/string.md)).
+
+## enabledProfiles {#enabled-profiles}
+
+ Returns settings profiles, assigned to the current user both explicitly and implicitly. Explicitly assigned profiles are the same as returned by the [currentProfiles](#current-profiles) function. Implicitly assigned profiles include parent profiles of other assigned profiles, profiles assigned via granted roles, profiles assigned via their own settings, and the main default profile (see the `default_profile` section in the main server configuration file).
+
+ If some profile occurs multiple times (with some other settings in between), the latest occurrence overrides all the previous ones.
```

Comment:
```suggestion
 The order of the profiles in the list corresponds to the order of applying these profiles.
```

Reply:
This sentence is describing kind of implementation details, so it's better to remove it.